### PR TITLE
numa.numa_stress: Add -p option for mkdir to avoid error if directory exists

### PR DESCRIPTION
**Pull request by huangyum**
Add -p option for mkdir to avoid error if directory exists, and remove it in finally.
ID: 1412083
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

**Comment by ngu-niny** [comment]
LGTM and test PASS:

'1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.numa.numa_stress' Finshed 2017-01-11 05:41:51 2017-01-11 06:56:27 PASS 4475.407563
'2-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.6.9.x86_64.io-github-autotest-qemu.numa.numa_stress' Finshed 2017-01-11 06:56:30 2017-01-11 08:12:04 PASS 4534.313376
'3-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.numa.numa_stress' Finshed 2017-01-11 08:12:09 2017-01-11 09:27:20 PASS 4510.962994
'4-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.numa.numa_stress' Finshed 2017-01-11 09:27:23 2017-01-11 10:41:18 PASS 4434.962048

Acked-by: Nini Gu <ngu@redhat.com>

File: qemu/tests/numa_stress.py [hunk 1]
```diff
@@ -41,16 +41,16 @@ def get_tmpfs_write_speed():
     Get the tmpfs write speed of the host
     return: The write speed of tmpfs, the unit is kb/s.
     """
-    utils.run("mkdir /tmp/test_speed && mount -t tmpfs none /tmp/test_speed")
+    utils.run("test -d /tmp/test_speed || mkdir /tmp/test_speed && mount -t tmpfs none /tmp/test_speed")
```
Comment (CongLi) [inline qemu/tests/numa_stress.py]: Prefer add "-p" option for mkdir.

Comment (huangyum) [inline qemu/tests/numa_stress.py]: The /tmp always exists,  "-p" option seems useless here.

Comment (CongLi) [inline qemu/tests/numa_stress.py]: no error if dir is existing.

Comment (huangyum) [inline qemu/tests/numa_stress.py]: You are right, thanks, updated!

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

File: qemu/tests/numa_stress.py [hunk 2]
```diff
@@ -41,16 +41,16 @@ def get_tmpfs_write_speed():
     Get the tmpfs write speed of the host
     return: The write speed of tmpfs, the unit is kb/s.
     """
-    utils.run("mkdir /tmp/test_speed && mount -t tmpfs none /tmp/test_speed")
+    utils.run("mkdir -p /tmp/test_speed && mount -t tmpfs none /tmp/test_speed")
     output = utils.run("dd if=/dev/urandom of=/tmp/test_speed/test bs=1k count=1024")
```
Comment (yanan-fu) [inline qemu/tests/numa_stress.py]: From the title: "check if /tmp/test_speed exists"    where is the check ?

**Comment by xutian** [comment]
looks ok to me

**Pull request merged**